R/checks.R

```diff
@@ -201,9 +201,13 @@ check_installs <- function(x) {
   if (length(deps) > 0) {
     is_inst <- purrr::map_lgl(deps, is_installed)
     if (any(!is_inst)) {
-      rlang::abort(c("Some package installs are required: ",
-                     paste0("'", deps[!is_inst], "'", collapse = ", ")
-      ))
+      deps_not_installed <- deps[!is_inst]
+      deps_not_installed <- deps_not_installed[!duplicated(deps_not_installed)]
```

Comment:
```suggestion
      deps_not_installed <- unique(deps_not_installed)
```

Isn't this a fancy way to do `unique()`?

Reply:
Heard! [d6b958f](https://github.com/tidymodels/tune/pull/727/commits/d6b958f5c3a803b5ca4ef6af484ac74f377651db).